index.rst

```diff
@@ -4,22 +4,215 @@
+
+When required to be under stricter change controls, branch protection should be put in place.
+The designation of specific reviewers can also be manually specified.
+In the event that this requires higher-level SIT-Com repo permissions, please consult the ``#rubinobs-sitcom-docs`` Slack channel for support.
+
+General Documentation Types
+===========================
+
+This section outlines the different types of documentation used by SIT-Com personel.
+
+Jira Tickets
+^^^^^^^^^^^^
+
+SIT-Com related work should follow the `SIT-Com specific Jira workflow <https://sitcomtn-023.lsst.io/>`_.
+Documentation in Jira tickets should be sufficiently detailed such that it addresses the description in the task.
+It is not expected to be consulted in the future, especially on a regular occurrence. 
+In many cases, specifically when characterization activities have taken place over a series of tickets, external documentation should be developed and linked to the ticket(s).
+The type of document to be created is case dependent, but will most likely be covered by `Technotes`_ and/or inclusion of a section in the `Obs-Ops Webspace`_.
+The creation of the documentation can be performed under one of the already opened tickets, or, a new ticket can be created that explicitly deals with the generation of documentation.
+
+
+Verification Documentation and Artifacts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Systems Engineering team uses a specific Jira project for its verification performance and tracking.
+Personnel looking for verification documentation should consult `LSE-160 <ls.st/LSE-160>`_.
+
+The verification and validation process also utilize a specific repository for the storing of their notebooks that act as official artifacts.
+This is further described in the `Verification And Validation Notebooks`_ subsection below.
+
+
+Technotes
+^^^^^^^^^
+
+Technotes are useful for documenting outcomes of specific tests and/or findings, proposing new functionality or architectures, and/or documenting specific usages that augment user documentation.
+They can also be used as logs and/or status of longer term analyses.
+An example of this is the technote on Building and and fitting the pointing model for the AuxTel (`TSTN-014 <https://tstn-014.lsst.io/>`_.)
+In this case, there are chapters (seen on the left) showing the analysis from different runs and the results that were obtained.
+This is an appropriate use of a technote.
+
+Technotes should not be used in place of user-facing how-to documentation and/or procedures. 
+This is better addressed as part of the `Obs-Ops Webspace`_ or even an addition to the collection of `Collaborative Notebooks`_.
+
+Creation and Review of Technotes
+--------------------------------
+
+When creating a technote, a Jira ticket should be created following the `SIT-Com specific Jira workflow <https://sitcomtn-023.lsst.io/>`_, this is primarily to facilitate an easier review process and to track progress.
```

Comment:
This phrase reads a bit weird. It has "creating" ... "created" in the first part and in the second part the "facilitate an easier"  seems a bit redundant. Maybe something like:

> When working on a new technote, or editing an existing one, start by creating a Jira ticket in the `SIT-Com specific Jira workflow <https://sitcomtn-023.lsst.io/>`_, this allows us to track the work and facilitates the review process.

Reply:
Good point. I didn't use your example but I did modify it.